2 дз

ilya-cell opened the pull request:
типо разметка. я ещё пытался иконку сделать, но она как-то не отображается(

ilya-cell committed:
Create index.html

ilya-cell committed:
второе дз

типо разметка. я ещё пытался иконку сделать, но она как-то не отображается(

MitrofanovaO commented on index.html:
```diff
@@ -0,0 +1,146 @@
+<head>
+    <meta charset="UTF-8">
+    <title>Main page</title>
+    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
```
С неё всё нормально, она отображается 
![image](https://user-images.githubusercontent.com/24611929/118176770-e3f11180-b442-11eb-95f0-ca0c8df47343.png)

MitrofanovaO commented on index.html:
```diff
@@ -0,0 +1,146 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Main page</title>
```
в тайтле надо прописать название проекта (Design Life Kids)

MitrofanovaO commented on index.html:
```diff
@@ -0,0 +1,146 @@
+</head>
+<body>
+    <header class="header"> 
+            <img src="#" alt="logo">
```
логотип должен быть обернут в ссылку, потому что на внутренних страницах мы можем нажать на логотип и вернуться на главную

MitrofanovaO commented on index.html:
```diff
@@ -0,0 +1,146 @@
+                 <li><a href="#">ABOUT US</a></li>
+             </ul>
+        </nav>
+        <img src="#" alt="link">
+      <a href="#">CONTACT US</a>
```
отдельный блок если смотреть по верстке (должен быть объединен дивом одним) иконка search я б сделала кнопкой

MitrofanovaO commented on index.html:
```diff
@@ -0,0 +1,146 @@
+      <a href="#">CONTACT US</a>
+    </header>
+    <main>
+        <article>
```
у тебя везде смысловые блоки обернуты в article, тут лучше section там где есть заголовки - точно

MitrofanovaO commented on index.html:
```diff
@@ -0,0 +1,146 @@
+    </header>
+    <main>
+        <article>
+            <div class="image">
+                <img src="#" alt="bg" style="width:100%;">
+                <div class="top-left">
+                    <ul>
+                        <br>01
+                        <br><img src="#" alt="stripes">
+                    </ul> 
+                </div>
+                <ul>
+                    <div>THE OOH NOO 
+                        <br> COLLECTION</div>
+                        <div>Designed in Slovenia</div>
+                        <button>EXPLORE NOW</button>
+                </ul>
+                <div>Free (continental) USA shipping on all orders of $100 USD or more!</div>
+              </div>
```
тут должен быть размечен слайдер с пагинацией с колличеством слайдов равным количеству палочек (пагинаций) и числом (какой сейчас активный слайд)

MitrofanovaO commented on index.html:
```diff
@@ -0,0 +1,146 @@
+        <article>
+            <h2>Category Products</h2>
+            <nav>
+                <ul>
+                <div class="goods"> 
+                    <img src="#" alt="sweater">
+                    <div>APPAREL
+                        KIDS</div>
+                </div>
+                <img src="#" alt="sock">
+                <div>HOLIDAY SHOP</div>
+                <img src="#" alt="deer">
+                <div>TOYS</div>
+                <img src="#" alt="bigfoot">
+                <div>ACCESSORIES</div>
+                </ul>
```
валиданию не прогонял? тут будет ошибка, в ul не может быть div, img и других селекторов. В ul детьми могут быть только li, а уже в li может быть всё что угодно

MitrofanovaO commented on index.html:
```diff
@@ -0,0 +1,146 @@
+                <div>ACCESSORIES</div>
+                </ul>
+            </nav>
+            <button><img src="#" alt="back"></button>
+            <button><img src="#" alt="forward"></button>
+            <h6>Modern finds for the modern home. Curated goods by unique designers around the world from fashion to furniture for kids and adults.</h6>
```
это всё относится так же к слайдеру

MitrofanovaO commented on index.html:
```diff
@@ -0,0 +1,146 @@
+                <a href="#">2018 Holiday Catalog</a>
+                <a href="#">Shop By Designer</a>
+            </ul>
+            <img src="#" alt="line">
```
линию будем делать с помощью css, так что везде их можешь убрать

MitrofanovaO commented on index.html:
```diff
@@ -0,0 +1,146 @@
+            <img src="#" alt="line">
+            <input type="email">
+            <button>Join</button>
+            <ul>
+                <img src="#" alt="logo">
+                <img src="#" alt="logo">
+                <img src="#" alt="logo">
+                <img src="#" alt="logo">
+            </ul>
```
только ul>li, никак больше, везде надо добавить картинки, так очень непонятно что сделано а что нет

MitrofanovaO commented on index.html:
```diff
@@ -0,0 +1,146 @@
+            <img src="#" alt="logo">
+            <img src="#" alt="logo">
+        </article>
+        <article>
```
здесь я бы добавила общий див для эти двух article (потому что они одинаковые по 50% от блока), чтобы дальше можно было стилизовать

MitrofanovaO commented on index.html:
```diff
@@ -0,0 +1,146 @@
+            <div>
+                <img src="#" alt="bg">
+                <h2>Design Life Kids</h2>
+                <h4>About Us</h4>
```
это обычный спан или див, не ставила бы тут заголовок, у тебя уже есть тут заголовок h2 в блоке, а это просто текст

MitrofanovaO commented on index.html:
```diff
@@ -0,0 +1,146 @@
+                <a href="#">Shop By Designer</a>
+            </ul>
+            <img src="#" alt="line">
+            <a href="#">Company Rating</a>
+            <img src="#" alt="5stars">
```
это объединить в один блок и сделать список из звездочек ul>li

ilya-cell committed:
personal

вставил часть картинок, позже продолжу

ilya-cell committed:
Revert "personal"

This reverts commit b48ddfc8cf406a59b06cf6a9443c6e36fa874903.

ilya-cell committed:
ну типа

ilya-cell committed:
Вставил часть картинок дубль 2

ilya-cell committed:
что-то

биип-биип-биип выгружаааю

ilya-cell committed:
save

*turn computer off

ilya-cell committed:
разметка

ilya-cell committed:
сейф (разводящий руками смайлик)

поубирал пробелы, заглавные буквы, создал файлики для css (теперь надо бы их заполнить...) распихал всё по папочкам

MitrofanovaO commented on index.html:
```diff
@@ -0,0 +1,144 @@
+        </a>
+        <nav>
+             <ul>
+                <li>
+                    <a href="#">blog</a>
+                    <a href="#">catalog</a>
+                    <a href="#">about us</a>
+                </li>
```
каждый пункт меню должен быть обернут в li

MitrofanovaO commented on index.html:
```diff
@@ -0,0 +1,144 @@
+      <a href="#">contact us</a>
+    </header>
+    <main>
+        <section class="image">
+                <img src="images/imagemonster.png" alt="monster">
+                <h1>01</h1>
+                <ul>
+                    <div>the oh noo collection</div>
+                    <div>Designed in Slovenia</div>
+                    <a href="#"><button>explore now</button></a>
+                </ul>
+                <div>Free (continental) USA shipping on all orders of $100 USD or more!</div>
+        </section>
```
секция должна быть с классом slider (мы уже проговорили что это листающаяся такая чтучка(с) ) 01 это не главнй заголовок h1 страницы (заголовок страницы находится наверху страницы и  у него зачастую наибольший размер шрифта, а теперь вопрос где у тебя заголовок на странице если смотреть на макет) + у тебя нет в этом блоке пагинации (4 плочки на макете под цифрой 01) + ul всегда должен состоять из li (у тебя же в ul стоят div и a, это ошибка) нужно всемто ul поставить div. + button не должен быть обернут в ссылку, не поняла зачем ты это сделал.
Нужно скомпоновать H1 + div button + img + div в один див, так называемый slide + надо объединить пагинацию и вынести за пределы этого слайда, таких слайдов должно быть 4 шт (столько сколько у тебя в пагинациях блочков - по макету 4 (1 активный слайд и 3 мы не видим)), тоесть тебе надо будет продублировать первый слайд ещё 3 раза и потом навесить класс hidden и скрыть их со станицы (почему мы так делаем спросишь ты, по одному из критериев должна присутствовать вся разметка html, а если мы не разметим остальные 3 слайда - косяк, минус баллы)

MitrofanovaO commented on index.html:
```diff
@@ -0,0 +1,144 @@
+        </section>
+        <section>
+            <h1>Category Products</h1>
+            <nav>
```
тег навигации здесь не уместен, это не меню сайта, это категории

MitrofanovaO commented on index.html:
```diff
@@ -0,0 +1,144 @@
+            <h1>Category Products</h1>
+            <nav>
+                <ul>
+                    <li>
+                        <div class="goods"> 
+                        <a href=""><img src="images/category-products/imagesweater.png" alt="sweater"></a>
+                        <div>apparel kids</div>
+                        </div>
+                        <a href=""><img src="images/category-products/imagesock.png" alt="sock"></a>
+                        <div>holiday shop</div>
+                        <a href=""><img src="images/category-products/imagedear.png" alt="deer"></a>
+                        <div>toys</div>
+                        <a href=""><img src="images/category-products/imagebigfoot.png" alt="bigfoot"></a>
+                        <div>accessoires</div>
+                    </li>
```
неправильняа структура, у тебя должен идти список ul<li , в li ссылка a, в которой находится картика и заголовок, количесство li будет равное количеству блоков

MitrofanovaO commented on index.html:
```diff
@@ -0,0 +1,144 @@
+            </nav>
+            <button><img src="images/arrows/vectorback.svg" alt="back"></button>
+            <button><img src="images/arrows/vectorforvard.svg" alt="forward"></button>
+            <h6>Modern finds for the modern home. Curated goods by unique designers around the world from fashion to furniture for kids and adults.</h6>
```
это не заголовок h6 это просто текст к блоку (обычный обзац)

MitrofanovaO commented on index.html:
```diff
@@ -0,0 +1,144 @@
+        <section>
+            <img src="images/imagewoman.png" alt="woman">
+            <div class="order">
+                <ul>
+                    <li>
```
здесь ul<li не уместны, их лучше сделать здесь 
![image](https://user-images.githubusercontent.com/24611929/123138288-e7c27b80-d465-11eb-9cf3-7e6c1f88c7ee.png)

MitrofanovaO commented on index.html:
```diff
@@ -0,0 +1,144 @@
+    </main>
+    <footer>
+        <div>
+            <h6>Navigation</h6>
```
лучще сделать тут h3 и везде где в футере подобные заголовки

MitrofanovaO commented on index.html:
```diff
@@ -0,0 +1,144 @@
+        </div>
+        <div>
+            <h6>Connect</h6>
+            <input type="email">
+            <button>Join</button>
```
здесь должен быть тег form в нем должны быть связанные между сбой label и input с помощью id и for + для input прописать placeholder и для button прописать type

MitrofanovaO commented on index.html:
```diff
@@ -0,0 +1,144 @@
+            <h6>Connect</h6>
+            <input type="email">
+            <button>Join</button>
+            <ul>
+                <li>
+                    <a href=""><img src="images/social-media/facebook.svg" alt="logo"></a>
+                    <a href=""><img src="images/social-media/fill11twitter.svg" alt="logo"></a>
+                    <a href=""><img src="images/social-media/pinterest.svg" alt="logo"></a>
+                    <a href=""><img src="images/social-media/instagram.svg" alt="logo"></a>
+                </li>
+            </ul>
```
опять же  неправильняа структура, у тебя должен идти список ul<li , в li ссылка a, в которой находится картика и заголовок, количество li будет равное количеству блоков

ilya-cell committed:
save

выключал комп

ilya-cell committed:
raiting

сделал норм рейтинг

ilya-cell committed:
...
